Assets/unZENity-VR/Scripts/Settings/SettingsManager.cs

```diff
@@ -23,19 +23,26 @@ private void LoadGameSettings()
         {
             var settingsFilePath = $"{GetRootPath()}/{SETTINGS_FILE_NAME}";
             if (!File.Exists(settingsFilePath))
-                throw new ArgumentException($"No >GameSettings.json< file exists at >{settingsFilePath}<. Can't load Gothic1.");
+                if (Application.platform == RuntimePlatform.Android)
+                    Importer();
```

Comment:
We still need to throw an exception.

Because by default our game doesn't know where to look for the Gothic installation.
Therefore let's throw the error and tell user "what to do next".

Reply:
I understand now: If the file doesn't exist, just create it. True!